pkg/install/upgrade.go

```diff
@@ -20,18 +22,20 @@ func (i *Installer) upgradeCluster(ctx context.Context) error {
 			return err
 		}
 
+		// We don't upgrade if cluster upgrade is not finished
+		if !status.ClusterVersionOperatorIsHealthy(cv.Status) {
+			return fmt.Errorf("not upgrading: previous upgrade in-progress")
```

Comment:
:+1:

Reply:
Is this error message indicative of only of a `previous upgrade in-progress`?

Reply:
most cases yes. Either way, this will require manual checking